es6-shim.js

```diff
@@ -205,7 +205,15 @@
     return result;
   };
 
+  var bindContext = function (func, context) {
+    return function () {
+      return func.apply(context, arguments);
+    };
+  };
+
   var ES = {
+    Call: bindContext(Function.prototype.call, Function.prototype.apply),
```

Comment:
Good call - https://people.mozilla.org/~jorendorff/es6-draft.html#sec-call-f-v-args is in section 7, so this is good.